packages/video_player/pubspec.yaml

```diff
@@ -7,14 +7,23 @@ homepage: https://github.com/flutter/plugins/tree/master/packages/video_player
 
 flutter:
   plugin:
-    androidPackage: io.flutter.plugins.videoplayer
-    iosPrefix: FLT
-    pluginClass: VideoPlayerPlugin
+    platforms:
```

Comment:
We can't land this before next stable release as the plugin won't compile with the stable tool.